test/integration/consul-container/libs/consul-cluster/consul-cluster.go

```diff
@@ -0,0 +1,105 @@
+package consul_cluster
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	consulnode "github.com/hashicorp/consul/integration/consul-container/libs/consul-node"
```

Comment:
Similarly, if this package was called `node` we wouldn't need the import alias here 🙇🏻‍♂️